Abot2/Core/HyperLinkParser.cs

```diff
@@ -84,16 +84,14 @@ protected virtual IEnumerable<HyperLink> GetHyperLinks(CrawledPage crawledPage,
             if (!string.IsNullOrEmpty(baseHref))
             {
                 if (baseHref.StartsWith("//"))
-                    baseHref = crawledPage.Uri.Scheme + ":" + baseHref;
-
-                try
-                {
-                    uriToUse = new Uri(baseHref);
-                }
-                catch
-                {
-                    // ignored
-                }
+                    baseHref = uriToUse.Scheme + ":" + baseHref;
```

Comment:
I believe the crawledPage.Uri.Scheme is "more" correct here as we can't change the scheme of the crawl because the responding server returned a new scheme. It believe it would cause some confusion under that scenario.

Reply:
I would disagree. Since the responding server is the responding server, it should dictate how you're interpreting the base tag - just like your browser would if you were to visit the page.

Either way, as the change doesn't relate directly to any of the 2 issues that this PR aims to resolve, I'll remove the change.